js/values/Paper.js

```diff
@@ -1,15 +1,18 @@
 /**
  * Construct the Paper object and set the keyword property
- * @param {string} keyword
+ * @param {string} keyword The keyword to use for analyses.
+ * @param {object} metas The object containing all metavalues.
  * @constructor
  */
-var Paper = function( keyword ) {
+var Paper = function( keyword, metas ) {
```

Comment:
As far as I know, metas isn't a word. Suggestion: use metadata instead. This is automatically more descriptive as well.